zulipterminal/ui_tools/views.py

```diff
@@ -1127,4 +1134,4 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
         return key
 
     def set_spinner(self, spinner: str) -> None:
-        self.base_widget.set_text(self.text + [spinner])
+        self.body.base_widget.set_text(self.text + [spinner])
```

Comment:
Is this something necessary for this commit? Couldn't we move this to the previous commit where you introduce the `set_spinner` class?